homeassistant/components/ambient_station/strings.json

```diff
@@ -112,6 +118,24 @@
       "leak_detector_4": {
         "name": "Leak detector 4"
       },
+      "pm_in_humidity_aqin": {
+        "name": "PM indoor humidity (AQIN)"
+      },
+      "pm_in_temperature_aqin": {
+        "name": "PM indoor temperature (AQIN)"
+      },
+      "pm10_indoor_aqin": {
+        "name": "PM10 indoor (AQIN)"
+      },
+      "pm10_indoor_24h_average_aqin": {
+        "name": "PM10 indoor 24 hour average (AQIN)"
+      },
+      "pm25_indoor_aqin": {
+        "name": "PM2.5 indoor (AQIN)"
+      },
+      "pm25_indoor_24h_average_aqin": {
+        "name": "PM2.5 indoor 24 hour average (AQIN)"
+      },
```

Comment:
Are these values combined? So let's say I have a weatherstatio, and an AQIN, do they have to be close to each other? Or can I place them in different rooms? Like, are they 2 different devices.

Would it make sense to register this device as a different device in HA? This way you can assign them to different rooms and then we don't need to have that `aqin` suffix